plugins/module_utils/mysql.py

```diff
@@ -41,6 +45,9 @@ def mysql_connect(module, login_user=None, login_password=None, config_file='',
                   connect_timeout=30, autocommit=False, config_overrides_defaults=False):
     config = {}
 
+    if not HAS_MYSQL_PACKAGE:
+        module.fail_json(msg=missing_required_lib("PyMySQL"), exception=MYSQL_IMP_ERR)
```

Comment:
@Andersson007  I'm not sure about this line.  
Shall we keep only `PyMySQL` here? The self-written `MYSQL_IMP_ERR` includes both libraries.

Reply:
@markuman I'm also not sure. I wonder if we can use `PyMySQL|mysqlclient`  or something like that